Modules/Sources/Yosemite/Model/Bookings/FilterBookingsByProduct.swift

```diff
@@ -0,0 +1,20 @@
+// periphery:ignore:all - will be used for booking filters
+import Foundation
+
+/// Used to filter bookings by product
+///
+public struct FilterBookingsByProduct: Codable, Hashable {
```

Comment:
How come this is named Filter*, whereas BookingDateRangeFilter has Filter at the end? Based on the comments, the intentions are the same; if this is the case, I'd expect they would use the same naming convention.